pyp2rpm/convertor.py

```diff
@@ -4,17 +4,16 @@
 import sys
 try:
     import urllib2 as urllib
+    from urllib2 import urlopen, HTTPError, URLError
 except ImportError:
     import urllib
     from urllib import request
+    from urllib.request import urlopen, HTTPError, URLError
```

Comment:
These imports are an excellent target for improvement when python 2 support is removed.